docarray/index/backends/weaviate.py

```diff
@@ -358,6 +370,47 @@ def find(
 
         return FindResult(documents=docs, scores=scores)
 
+    def find_subindex(
```

Comment:
Why does weaviate need to implement its own `find_subindex` instead of using the one from the abstract class?

Reply:
Because the `search_field` for other backend is like `docs__simple_tens`, which includes the nested field name. But it should be only `docs` for weaviate.

> Weaviate is configured to allow one vector per data object